### DISCUSSION TITLE
How to do a catch all 404 page in App router?

### DISCUSSION THREAD

**Discussion by eRuaro** [vercel/next.js#53969] [App Router] [answered]
Is there a way to have a catch all 404 page similar to `react router`? Something like this:
```
{
	path: "*",
	element: <Page404/>
}
```

**Comment by icyJoseph** [comment, marked as answer]
- https://nextjs.org/docs/app/api-reference/file-conventions/not-found

> Good to know: In addition to catching expected notFound() errors, _**the root app/not-found.js file also handles any unmatched URLs for your whole application. This means users that visit a URL that is not handled by your app will be shown the UI exported by the app/not-found.js file.**_
